scripts/validate-schemas.js

```diff
@@ -7,82 +7,77 @@ const ajvDraft04 = new AjvDraft04();
 const Ajv = require('ajv');
 const ajv = new Ajv();
 
-function validation (excludedFiles){
-
+/* eslint-disable sonarjs/cognitive-complexity */
```

Comment:
@AnimeshKumar923 As you recently created this script, would you be happy improving it by reducing the cognitive complexity Sonarcloud complains at this moment? I guess it is a matter of simplifying the script.

If so, let's create a new issue for it.